vars/neoDeploy.groovy

```diff
@@ -186,6 +191,21 @@ def call(parameters = [:]) {
                           dockerEnvVars: configuration.get('dockerEnvVars'),
                           dockerOptions: configuration.get('dockerOptions')) {
 
+                          NEO_HOME_CHECK: {
+                              // same order like inside getNeoExecutable
+                              String neoHome = configuration.neoHome ?: env?.NEO_HOME
+
+                              // In case neo home is not set, but neo toolset is simply
+                              // in the path, we trust that everything is OK. In order to
+                              // validate the version also in this case, we need to adjust
+                              // toolValidate.
+
+                              def rc = sh script: "which ${NEO_DEFAULT_CMD}", returnStatus: true
+                              if(neoHome || (!neoHome && rc != 0)) {
```

Comment:
I would directly fail here if  (!neoHome && rc != 0) instead of passing an empty neohome ot toolValidate. This would be better readable/understandable.

Reply:
Furthermore some log output could be helpful if neo is taken from the path.

Reply:
We can discuss if we should fail here in case ```!neoHome```. With the current approach the behavior is closer to the behavior before the change. Before the change we got inside ```toolValidate```and fail inside in the same was like it is now.

Reply:
log output: done